app/frontend/packages/multiselect-dropdown/src/components/MultiselectDropdown.res

```diff
@@ -13,6 +13,8 @@ type searchItem = {
   text: string,
 }
 
+let tr = I18n.t(~scope="components.MultiselectDropdown")
+
```

Comment:
`I18n` is not part of the `MultiselectDropdown` package; this will break `MultiselectDropdown` when used outside the LMS.